Sources/Datadog/CrashReporting/DDCrashReportingPluginType.swift

```diff
@@ -146,13 +146,6 @@ public class DDCrashReport: NSObject {
     /// The last context injected through `inject(context:)`
     internal let context: Data?
 
-#if DD_SDK_ENABLE_INTERNAL_MONITORING
-    /// Additional diagnostic information about the crash report, collected for `DatadogCrashReporting` observability.
-    /// Available only if internal monitoring is enabled, disabled by default.
-    /// See: `Datadog.Configuration.Builder.enableInternalMonitoring(clientToken:)`.
-    public var diagnosticInfo: [String: Encodable] = [:]
```

Comment:
This was reported with all crashes in `dogfooding`, better to remove this payload with RUM Telemetry.